### PR TITLE
Update URLs mixin description with full sample

### PR DESCRIPTION
**Pull request by miggland**
The example there before is too minimal, IMHO..

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair happy with this extra detail?

**Comment by matmair** [comment]
There are unneeded infos and a pattern that will lead to errors in the proposed change. I can propose a larger example but I would not recommend this pattern.

**Comment by miggland** [comment]
The existing example is not complete. There is a function `url(` which seems wrong to me. There is not example of how the function `self.view_increase` should be implemented.

IMO, including which imports are necessary is quite important to get started with a plugin.

What will lead to errors?

File: docs/extend/plugins/urls.md [hunk 1]
```diff
@@ -8,13 +8,35 @@ Use the class constant `URLS` for a array of URLs that should be added to InvenT
 
 The array has to contain valid URL patterns as defined in the [django documentation](https://docs.djangoproject.com/en/stable/topics/http/urls/).
 
+Minimal sample:
+
```
Comment (matmair) [inline docs/extend/plugins/urls.md:12]: Suggestion: All samples are minimal; Full working examples are provided in the main code base so this is redundant.
```suggestion
```

File: docs/extend/plugins/urls.md [hunk 2]
```diff
@@ -8,13 +8,35 @@ Use the class constant `URLS` for a array of URLs that should be added to InvenT
...
+from plugin.mixins import UrlsMixin
+
+
+def start_page(request):
+
+    return HttpResponse("Hello, World!")
+
+
+class CustomPagerPlugin(UrlsMixin, InvenTreePlugin):
+    """Plugin to display custom pages"""
```
Comment (matmair) [inline docs/extend/plugins/urls.md:28]:
```suggestion
from django.urls import re_path
from django.http import HttpResponse

from plugin import InvenTreePlugin
from plugin.mixins import UrlsMixin


class CustomPagerPlugin(UrlsMixin, InvenTreePlugin):
    """Plugin to display custom pages."""
```

File: docs/extend/plugins/urls.md [hunk 3]
```diff
@@ -8,13 +8,35 @@ Use the class constant `URLS` for a array of URLs that should be added to InvenT
...
 
-    NAME = "UrlsMixin"
+    NAME = "Sample URL Plugin"
+    SLUG = "sampleurl"
+    TITLE = _("Sample URL Plugin")
+    DESCRIPTION = _("Sample URL Plugin")
+    VERSION = "0.0.1"
+    AUTHOR = _("Plugin Guy")
```
Comment (matmair) [inline docs/extend/plugins/urls.md:35]:
```suggestion

    def start_page(request):
        return HttpResponse("Hello, World!")
```
None of the additional meta fields are needed to get a working URLsMixin

File: docs/extend/plugins/urls.md [hunk 4]
```diff
@@ -8,13 +8,35 @@ Use the class constant `URLS` for a array of URLs that should be added to InvenT
...
 
     URLS = [
-        url(r'increase/(?P<location>\d+)/(?P<pk>\d+)/', self.view_increase, name='increase-level'),
+        # Start page
```
Comment (matmair) [inline docs/extend/plugins/urls.md:38]:
```suggestion
```

**Comment by miggland** [comment]
I'm sorry if my attempt to contribute annoyed you.

**Pull request closed without merging**